src/trainers/trainer-service.js

```diff
@@ -0,0 +1,50 @@
+// declare URL where server listens for HTTP requests
+const TRAINERS_URL = "http://localhost:8080/api/trainers"
+
+// retrieve all users from the server
+export const findAllTrainers = () =>
+    fetch(TRAINERS_URL)
+    .then(response => response.json())
+
+
+// retrieve a single Trainerby their ID
+export const findTrainerById = (id) =>
+    fetch(`${TRAINERS_URL}/${id}`)
```

Comment:
Forgot to mention this in the api docs, but make sure everything ends in a `/` (trailing slash) e.g.  ``${TRAINERS_URL}/${id}/`. Django doesn't like the non-trailing slash unless I specifically add a setting to do so, but that can come in a future PR